src/java/org/apache/cassandra/io/sstable/CQLSSTableWriter.java

```diff
@@ -20,13 +20,9 @@
 import java.io.Closeable;
 import java.io.IOException;
 import java.nio.ByteBuffer;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.NavigableSet;
+import java.util.*;
```

Comment:
unintended?

Reply:
good catch. Interestingly, optimize-import introduces the wildcard import.